mkdocs.yml

```diff
@@ -3,12 +3,20 @@ site_dir: 'rendered-docs'
 # Not having this (or viewing the site locally, or from any place other than the
 # site_url) leads to version switching failing to preserve the current path.
 site_url: 'https://jj-vcs.github.io/jj/'
+repo_url: https://github.com/jj-vcs/jj
+repo_name: jj-vcs/jj
+edit_uri: edit/main/docs/
 theme:
   name: 'material'
   language: 'en'
   favicon: images/favicon-96x96.png
   features:
     # - navigation.top
+    - content.action.edit
+
+  icon:
+    repo: fontawesome/brands/github
+    edit: material/pencil
```

Comment:
Changed from the default:

![image](https://github.com/user-attachments/assets/ef65cbe1-1eb3-4cd0-bb8c-65a488c11e4e)